develop.html.mako

```diff
@@ -112,6 +113,26 @@ requested changes are not responded towards will be closed.</li>
 <li>Please make sure you are configured to run the current <a href="#environment">Development Environment</a>
     requirements, which will help ensure your PRs pass automatic tests.</li>
 
+<li>Instead of editing the Changelog file itself, please adopt the following workflow:
+    <ul>
+        <li>Create a per-issue changelog file with the path
+            `/doc/build/changelog/unreleased_{Version}/{IssueId}.rst`, in which
+            `IssueId` is the auto-generated GitHub Issue Id.
+            </li>
+        <li>If the PR does not have a corresponding issue, use a placeholder file
+            in your PR to generate a unique identifier for the PR, and then rename
+            the placeholder file to use the PR identifer for the `IssueId`.
+            </li>
+        <li>The changelog file format will hopefully be self-documenting
```

Comment:
"The file format for a changelog file uses the .rst markup described at https://github.com/sqlalchemyorg/changelog .   See the README.txt file in the doc/build/unreleased_XX" directory for an example"

(then we have to add examples of the format in the README.txt files.... )